README.md

```diff
@@ -1 +1,23 @@
-# parasoft-demo-app-mobile
\ No newline at end of file
+# parasoft-demo-app-mobile
+The Parasoft Demo Application is an example Android application. The application is configurable, and is used to demonstrate functionality in a variety of Parasoft tools.
```

Comment:
Make the introduction more descriptive. 
Put explanation like the features this mobile app includes are subset of the web version (only approver).